src/Core/Framework/Store/Services/ExtensionLoader.php

```diff
@@ -246,6 +247,8 @@ private function loadLocalAppsCollection(Context $context): ExtensionCollection
                 'installedAt' => null,
                 'active' => false,
                 'type' => ExtensionStruct::EXTENSION_TYPE_APP,
+                'allowUpdate' => !$app->isManagedByComposer(),
```

Comment:
afaik apps can now also be required by composer :thinking: do we need to consider that here?

Reply:
yes that is what we do here ;) 

Apps can not use the `executeComposerCommands` so there managedByComposer is based on whether they are actually installed over composer

For plugins that flag got messed up as there could be plugins that are managed by composer, because `executeComposerCommands` is set to true, and therefore they are required, but they are still installed over custom/plugins